apps/fishing-map/features/i18n/LanguageToggle.tsx

```diff
@@ -26,6 +26,24 @@ const LanguageToggle: React.FC<LanguageToggleProps> = ({
   className = '',
 }: LanguageToggleProps) => {
   const { i18n } = useTranslation()
+  const [isLanguageMenuOpen, setIsLanguageMenuOpen] = useState(false)
```

Comment:
I think this is a great idea! but I think we should also close it when selecting a language